lib/zclient.h

```diff
@@ -1158,7 +1158,6 @@ zclient_send_rnh(struct zclient *zclient, int command, const struct prefix *p,
 		 vrf_id_t vrf_id);
 int zapi_nexthop_encode(struct stream *s, const struct zapi_nexthop *api_nh,
 			uint32_t api_flags, uint32_t api_message);
-extern int zapi_redistribute_stream_size(struct zapi_route *api);
```

Comment:
oh man - I'm sorry to say I just registered what you've done here.
The purpose of this function was to compute a _smaller_ stream buffer size, to avoid using a large buffer in a situation where that led to an excessive memory footprint. That's sort of the reverse of your case, where you want a really big buffer to satisfy your requirement in a different code path. I think you should leave this unchanged, and only change your own path at this time.

Reply:
yes, I think you're right: looking back at the rnh/nht change, it should be ok to take this change - sorry for the false alarm!